### ISSUE TITLE
Missing type hints

### ISSUE THREAD

**Issue by michalploomo** [dmitrymatlah/predictions-api#7]
In whole projects, type hints are missing.

**Comment by dmitrymatlah** [comment]
Please specify which variant of type hinting do you mean?
For ex. 
We have 

`     /**
     * PUT /api/v1/predictions/:id/status
     * allows to set status of the given prediction
     *
     * @param UpdatePredictionStatusRequest $request
     * @param Predictions $prediction
     * @return \Illuminate\Http\JsonResponse
     */
    public function updateStatus(UpdatePredictionStatusRequest $request, Predictions $prediction)`

`    /**
     * Determine if the validation rule passes.
     *
     * @param string $attribute
     * @param mixed $value
     * @return bool
     */
    public function passes(string $attribute, $value)
    {`

Or You about methods return type hinting

I didn't quite understand, why the wording is in the whole project?

**Comment by michalploomo** [comment]
Yes, I'm suggesting using method typing. In composer.json you are specifying "php": "^7.3|^8.0", so you can use code typehinting.
